spec/spec.md

```diff
@@ -573,67 +578,6 @@ This method can be used to enumerate the capabilities and limits of a wallet end
 
 Privileged methods require the current session to be in an authorized state to invoke them. For details on how a session enters and exits an authorized state, see the [non-privileged methods](#non-privileged-methods).
 
-#### sign_transactions
-
-##### JSON-RPC method specification
-
-###### Method
-{: .no_toc }
-
-```
-sign_transactions
-```
-
-###### Params
-{: .no_toc }
-
-```
-{
-    “payloads”: [“<transaction>”, ...],
-}
-```
-
-where:
-
-- `payloads`: one or more base64-encoded transaction payloads to sign
-
-###### Result
-{: .no_toc }
-
-```
-{
-    “signed_payloads”: [“<signed_transaction>”, ...],
-}
-```
-
-where:
-
-- `signed_payloads`: the corresponding base64-encoded signed transaction payloads
-
-###### Errors
-{: .no_toc }
-
-- `-32602` (Invalid params) if the params object does not match the format defined above
-- `ERROR_AUTHORIZATION_FAILED` if the current session is in the unauthorized state, either because [`authorize`](#authorize) has not been invoked for the current session, or because the current session's authorization has been revoked by the wallet endpoint
-- `ERROR_INVALID_PAYLOADS`
-
-  ```
-  “data”: {
-      “valid”: [<transaction_valid>, ...],
-  }
-  ```
-
-  if any transaction does not represent a valid transaction for signing, where:
-
-    - `valid`: an array of booleans with the same length as payloads indicating which are valid
-
-- `ERROR_NOT_SIGNED` if the wallet endpoint declined to sign these transactions for any reason
-- `ERROR_TOO_MANY_PAYLOADS` if the wallet endpoint is unable to sign all transactions due to exceeding implementation limits. These limits may be available via [`get_capabilities`](#get_capabilities).
-
-##### Description
-
-The wallet endpoint should attempt to simulate the transactions provided by data and present them to the user for approval (if applicable). If approved (or if it does not require approval), the wallet endpoint should sign the transactions with the private keys for the requested authorized account addresses, and return the signed transactions to the dapp endpoint.
-
 #### sign_and_send_transactions
```

Comment:
Spec for `sign_transactions` removed but spec for `sign_and_send_transactions` not added?

Reply:
sign_and_send_transactions was already in the spec

Reply:
the sign and send spec might need to be updated, however